scripts/qa_crowbarsetup.sh

```diff
@@ -2541,6 +2546,21 @@ function custom_configuration
                 $p "$l['tls_req_cert']" "'demand'"
                 $p "$l['use_pool']" "true"
             fi
+            if [[ $want_openid_connect = 1 ]] ; then
+                local p="proposal_set_value keystone default"
+                local f="['attributes']['keystone']['federation']"
```

Comment:
Why do you need the $f variable? Why not just define o as:

    local f="['attributes']['keystone']['federation']]'openidc']"

Reply:
Yes. I was leaving it for the possibility that we may need to set the attributes under "federation" as well. Was thinking of the future. :-) But I can combine it for now, till we really need it.